src/Illuminate/Support/Fluent.php

```diff
@@ -46,6 +46,29 @@ public function get($key, $default = null)
     }
 
     /**
+     * Set the value of an attribute.
+     *
+     * @param  string  $key
+     * @param  mixed   $value
+     * @return void
+     */
+    public function set($key, $value)
+    {
+        $this->attributes[$key] = $value;
```

Comment:
🤔 actually, maybe I should call `offsetSet` in `__set` and remove this method altogether. You may want to have 'set' attribute. The same with `has` which I introduced. I think this is why originally this class only had minimal set of methods.

The point of Fluent class was to use dynamic method calls:
`$fluent->set(1)->has(2) // ['set' => 1, 'has' => 2]`